extension_helpers/tests/test_setup_helpers.py

```diff
@@ -342,3 +342,95 @@ def get_extensions():
                 pass
             else:
                 raise AssertionError(package_name + ".compiler_version should not exist")
+
+
+@pytest.mark.parametrize("pyproject_use_helpers", [None, False, True])
+def test_only_pyproject(tmpdir, pyproject_use_helpers):
+    """
+    Test that makes sure that extension-helpers can be enabled without a
+    setup.py and without a setup.cfg file.
+    """
+
+    pytest.importorskip("setuptools", minversion="62.0")
+
+    package_name = "helpers_test_package_" + str(uuid.uuid4()).replace("-", "_")
+
+    test_pkg = tmpdir.mkdir("test_pkg")
+    test_pkg.mkdir(package_name).ensure("__init__.py")
+
+    simple_pyx = test_pkg.join(package_name, "simple.pyx")
+    simple_pyx.write(
+        dedent(
+            """\
+        def test():
+            pass
+    """
```

Comment:
```suggestion
        """\
        def test():
            pass
        """
```

Reply:
As we use black to deal with formatting I won't commit this suggestion for now but will look separately at why the formatting isn't quite right here.